src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -26,6 +26,21 @@ namespace Microsoft.PowerShell.EditorServices.Protocol.Server
 {
     public class LanguageServer
     {
+        private const string CheckHelpScript = @"
+[CmdletBinding()]
+param (
+    [String]$CommandName
+)
+Try {
+    $null = Microsoft.PowerShell.Core\Get-Command $CommandName -ErrorAction Stop
```

Comment:
If this script is only used in the `HandleShowOnlineHelpRequest` method, then it should be moved inside that method as a `const string`.

Reply:
This seems to make sense. As was said in slack: if it's a `const string` it won't be stack alloc'd each time the methos is invoked. https://stackoverflow.com/questions/7616448/const-string-inside-a-function

Unless @tylerl0706 @SeeminglyScience or @rjmholt have any objections I'll move the string back into the function as `const`...

Reply:
Reasonable to me! Let's hear what 1 other has to say

Reply:
Yep no issues with that 👍

Reply:
Moved the string into the function as const

(also creeped the scope a bit by handling both commands that could come in (although not from vscode-powershell...) and marking one as deprecated.